src/ui/tutorial/tutorial.js

```diff
@@ -337,7 +337,7 @@ cwc.ui.Tutorial.prototype.startTutorial = function() {
           youtube_videos: (step.videos || []).map((video) =>
             video['youtube_id']
           ),
-          hasCode: step.code && step.code.trim().length > 0,
+          hasCode: !!(step.code && step.code.trim().length),
```

Comment:
Could you please replace the "!!" with the actual condition like !goog.string.isEmptyOrWhitespace(...) or so.

Reply:
Done in latest commit.